os/board/common/README.md

```diff
@@ -0,0 +1,78 @@
+# How to partition a flash
+
+This document provides a way to partition in flash. Use CONFIG when flash partition.  
+
+## How to configure CONFIG
+
+Flash partition is provided by configurations, FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME. 
+FLASH_PART_SIZE represents size of each partition, FLASH_PART_SIZE does name, FLASH_PART_TYPE does interface between flash and upper layer.  
+All partition information is separated by comma and the last must end with a comma. The information of FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME should be in order. 
+
+ex)
+
+FLASH_PART_SIZE "16,48,192,32,512,2400,1536,1536,1400,8,512,"  
+FLASH_PART_TYPE "none,none,none,none,none,none,none,none,smartfs,config,none,"  
+FLASH_PART_NAME "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"  
+
+name    | type    | size
+------- | ------- | --------
+bl1     | none    | 16
+sssro   | none    | 48
+bl2     | none    | 192
+sssfw   | none    | 512
+wlanfw  | none    | 2400
+os      | none    | 1536
+factory | none    | 1536
+user    | smartfs | 1400
+nvram   | config  | 8
+sssrw   | none    | 512
+
```

Comment:
Add a sentence here "In the following, we describe how to set the parameters using menuconfig."

Reply:
Okay. I modified it.